### PR TITLE
Activity 3 - Online Resume of Bryan Suello

### PR DESCRIPTION
**Pull request by bryansuello**

### CODE REVIEW

File: Bryan_Suello/activity3/img/act3.png:Zone.Identifier [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]
```
Comment (mrmartincarlito) [inline Bryan_Suello/activity3/img/act3.png:Zone.Identifier:1]: Please remove this file

**Comment by mrmartincarlito** [comment]
Please remove all your Zone Identifier config,

File: Bryan_Suello/activity3/img/act3.png:Zone.Identifier (hunk 1 above)
Comment (bryansuello) [reply Bryan_Suello/activity3/img/act3.png:Zone.Identifier]: all similar files removed

**Comment by bryansuello** [comment]
all zone identifiers removed

**Comment by mrmartincarlito** [comment]
All good,
Thanks and very good!

**Pull request merged**